src/v/cluster/partition_probe.cc

```diff
@@ -267,6 +267,142 @@ void replicated_partition_probe::setup_public_metrics(const model::ntp& ntp) {
               .aggregate({sm::shard_label, partition_label}),
           });
     }
+
+    setup_public_scrubber_metric(ntp);
+}
+
+void replicated_partition_probe::setup_public_scrubber_metric(
+  const model::ntp& ntp) {
+    namespace sm = ss::metrics;
+
+    // No point in setting up the scrubber metrics if there's no
+    // archival metadata STM to pull values from.
+    if (!_partition.archival_meta_stm()) {
```

Comment:
Do we need to check for `config::shard_local_cfg().disable_public_metrics()`?

Reply:
`setup_public_scrubber_metrics` is called from `setup_public_metrics` which checks the cluster config.